src/cpp/DQM/VisDQMRenderTools.h

```diff
@@ -375,6 +376,11 @@ static string statWithErrorToJson(const TH2* const hist, const char* const name)
       .arg(name);
 }
 
+static string rootObjectToArray(const TObject* const h)
```

Comment:
Why did you choose this name? It implies that the returned value is an array... Is that true? Even if it is, the main focus is on the JSON part of it. Something like rootObjectToJSON would be preferable, in my opinion.